hooks/post-checkout

```diff
@@ -23,16 +24,35 @@ if [[ -n "${BUILDKITE_PLUGIN_TRIVY_SEVERITY:-}" ]] ; then
   export args+=("--severity ${BUILDKITE_PLUGIN_TRIVY_SEVERITY}")
   echo "using non-default severity types"
 fi
+
+if [[ -n "${BUILDKITE_PLUGIN_TRIVY_SECURITY_CHECKS:-}" ]] ; then
+  export fsargs+=("--security-checks ${BUILDKITE_PLUGIN_TRIVY_SECURITY_CHECKS}")
+  echo "using $BUILDKITE_PLUGIN_TRIVY_SECURITY_CHECKS security checks"
+else
+  echo "using default security checks"
+  export fsargs+=("--security-checks vuln,config")
```

Comment:
I think you can replace the two instances of `export fsargs+=(` with `fsargs+=(`. Are there any child processes that look for these variables?

Reply:
By default trivy does vulnerability(`vuln`) security checks but we decided to go with `vuln,Config` has a default security check. At the same time we decided to give users an option to pass a security check as a parameter. That's the reason we are appending `export fsargs+=(` with `fsargs+=(` with if and else condition it will be used in later `fs` scan.

Reply:
Hmm, I decided to try this in `/bin/bash` on macOS, and it does not like the use of `export` at all:

```
bash-3.2$ fsargs=()
bash-3.2$ export fsargs+=("--security-checks fOO")
bash: export: `fsargs+': not a valid identifier
bash-3.2$ fsargs+=("--security-checks fOO")
```

Reply:
Looks like it's an issue with Mac's bash, It working on Ubuntu's bash, and tests fine.
Also, shell check are not complaining.

Reply:
removed `export` and please check now (sometimes shell works automatically) anyways  I did the code changes.